integration_test/ginkgo_integration_test.go

```diff
@@ -0,0 +1,61 @@
+package integrationtest
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/node-disk-manager/integration_test/minikube_adm"
+	"github.com/openebs/node-disk-manager/integration_test/ndm_util"
+)
+
+func TestIntegrationNDM(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Integration Test Suit")
+}
+
+var _ = BeforeSuite(func() {
+	// It starts minikube if it is not Running
+	minikubeadm.Setup()
+
+	// It waits till namespace is ready
+	ndmutil.WaitTillDefaultNSisReady()
+
+	// It applies our CRDs
+	ndmutil.ApplyCRDs()
```

Comment:
Is this still required? Doesn't the operator.yaml include the required CRDs?

Reply:
Sorry I didn't noticed that CRDs are already there in ndm-operator.yaml. We can remove it now.